src/zenml/cli/utils.py

```diff
@@ -970,6 +970,42 @@ def print_components_table(
     print_table(configurations)
 
 
+def print_debug_stack(
+    stack: "StackResponseModel", components: "List[ComponentResponseModel]"
+) -> None:
+    """Print stack and components for debugging purposes.
```

Comment:
I think it would be nicer if this function extracted the components itself instead of them being passed as a separate argument